packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/bot_test.bicep

```diff
@@ -0,0 +1,97 @@
+param botServiceName string
```

Comment:
why do we add `_test` suffix to the file name? if no special reason is it possible to remove the suffix?

Reply:
Fxied.